sklearn/ensemble/bagging.py

```diff
@@ -487,6 +487,22 @@ class BaggingClassifier(BaseBagging, ClassifierMixin):
         was never left out during the bootstrap. In this case,
         `oob_decision_function_` might contain NaN.
 
+    Examples
+    --------
+
+    >>> from sklearn.model_selection import cross_val_score
+    >>> from sklearn.datasets import make_classification
+    >>> from sklearn.ensemble import BaggingClassifier
+    >>> from sklearn.svm import LinearSVC
+    >>> X, y = make_classification(random_state=0)
+    # Bagging ensemble of LinearSVC base estimators, each built on
+    # random subsets of 50% of the samples and 50% of the features.
+    >>> bagging = BaggingClassifier(LinearSVC(random_state=0),
+    ...                             max_samples=0.5, max_features=0.5,
+                                    oob_score=True)
```

Comment:
Is this a valid doctest? It seems that a continuation is missing in the beginning of the last line.